api/app/settings/common.py

```diff
@@ -460,3 +460,6 @@
 # Set this flag to prevent traits being stored for all Organisations within the application
 # Useful for data sensitive installations that dont want persistent traits.
 DEFAULT_ORG_STORE_TRAITS_VALUE = env.bool("DEFAULT_ORG_STORE_TRAITS_VALUE", True)
+
+# DynamoDB table name for storing environment
+DYNAMO_ENV_TABLE_NAME = env.str("DYNAMO_ENV_TABLE_NAME", None)
```

Comment:
Let's be explicit with the environment variable / setting. Something like: `ENVIRONMENTS_TABLE_NAME_DYNAMO`. `ENV` could be misunderstood.